src/components/SplitVertical/SplitVertical.spec.jsx

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import { shallow, mount } from 'enzyme';
 import assert from 'assert';
-import sinon from 'sinon';
 import _ from 'lodash';
 import { common } from '../../util/generic-tests';
 import SplitVertical from './SplitVertical';
 import DragCaptureZone from '../DragCaptureZone/DragCaptureZone';
 import { Motion } from 'react-motion';
 import { MOSTLY_STABLE_DELAY } from '../../../tests/constants';
 
+jest.setTimeout(10000);
```

Comment:
why?

Reply:
i'll try removing that too, i was running into the timeout